src/extension/webviews/connections_page/ConnectionEditor/BigQueryConnectionEditor/BigQueryConnectionEditor.tsx

```diff
@@ -54,13 +54,13 @@ export const BigQueryConnectionEditor: React.FC<
         </tr>
         <tr>
           <LabelCell>
-            <Label>Project ID:</Label>
+            <Label>Default Project ID:</Label>
           </LabelCell>
           <td>
             <TextField
-              value={config.projectName || ''}
-              setValue={projectName => {
-                setConfig({...config, projectName});
+              value={config.projectId || ''}
+              setValue={projectId => {
+                setConfig({...config, projectId});
```

Comment:
BTW, in https://github.com/malloydata/malloy-vscode-extension/pull/363 this is re-written in Lit, so depending on who lands first there will be merge conflicts somewhere.